### PR TITLE
feat: Signup added

### PR DESCRIPTION
**Pull request by ShoaibTamboli**
Sign up page created.

### CODE REVIEW

File: pages/login-signup/sign-up.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    padding: 0.3rem;
+}
+
+.button:hover{
+    background-color: var(--DBlue-color);
+    color: var(--white-color);
+}
+
+.input-textbox{
+    padding: 0.2rem;
```
Comment (sashaboi) [inline pages/login-signup/sign-up.css:39]: Maybe adding a little more padding, border radius and width parameters might make it look aesthetic?

File: pages/login-signup/login-signup.html [hunk 2]
```diff
@@ -50,21 +50,21 @@ <h2>Login</h2>
...
 
         <div>
 
             <label for="Password">Password</label> <br/>
-            <input type="password" name="Password" value="1234567890" >
+            <input type="password" name="Password" value="1234567890" class="input-textbox" >
 
         </div>
 
         <div class="label-holder"><input type="checkbox" name="Remember me"> <label for="Remember me" class="align-fields">Remember me</label> <span class="label-fpassword"><a href="#">Forgot your Password?</a></span></div>
```
Comment (sashaboi) [inline pages/login-signup/login-signup.html:63]: I am unclear on what the class align-fields does, maybe the styling for that is in an older version?

File: pages/login-signup/login-signup.html [hunk 3]
```diff
@@ -50,21 +50,21 @@ <h2>Login</h2>
...
             <label for="Email address">Email address</label> <br/>
-            <input type="text" name="Email address" placeholder="shoaibtamboli@gmail.com">
+            <input type="text" name="Email address" placeholder="shoaibtamboli@gmail.com" class="input-textbox">
         </div>
 
         <div>
 
             <label for="Password">Password</label> <br/>
-            <input type="password" name="Password" value="1234567890" >
+            <input type="password" name="Password" value="1234567890" class="input-textbox" >
```
Comment (sashaboi) [inline pages/login-signup/login-signup.html]: As per my knowledge , the password field need not have a value element ,might create bad UX

**Review by sashaboi — APPROVED** [review]
Please have a look at my comments, rest all looks good to go

File: pages/login-signup/login-signup.html (hunk 3 above)
Comment (sumitch29) [reply pages/login-signup/login-signup.html]: U can use placeholder here.

**Review by sumitch29 — APPROVED** [review]

**Pull request merged**